### DISCUSSION TITLE
allow user to select rct 2 TTT and rct deluxe and RCTC and let game optimize

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18089] [Feature requests] [answered]
I would like if we could select all 3 games to link all 3 installs in case we want more compact audio files, tycoon park, and for testing and debugging. Also could be in ini "location of rctc"

**Comment by Lastorder-DC** [comment, marked as answer]
RCTC already have all files for RCT2. No need to load from RCT2 if RCTC is connected.

**Reply by 733737** [reply]
it's not a need, it's a "see if this helps" kind of thing
